pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -403,3 +403,40 @@ func AddEnvironmentName() string {
 	handleError(err)
 	return strings.TrimSpace(environmentName)
 }
+func ApplicationOutputPath() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Provide a Path to where the Application is located",
```

Comment:
What do you think about using another phrase? This doesn't seem to match the init command, where we are starting from nothing. "Provide a path to where the application will be located" seems to be more appropriate.

Reply:
I have changed it in my next commit